### PR TITLE
feat(FE): version manager

### LINKED ISSUE #823: [Proposal]: Version management of route
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [ ] Bug
- [ ] Requirements
- [X] Feature or performance improvement
- [ ] Other

___
### Question
- What do you want to know?

___
### Bug
- Which version of Apache APISIX Dashboard, OS, and Browser?

- What happened?
If possible, provide a way to reproduce the error.

___
### Requirements or improvement
- Please describe your requirements or improvement suggestions.

Need more time to think.

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance

- Related issues
resolve #823 
___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: web/src/pages/Route/components/Step1/LabelsDrawer.tsx [hunk 1]
```diff
@@ -24,6 +24,8 @@ import { fetchLabelList } from '../../service';
 
 type Props = {
   title?: string;
+  labelPrefix?: string;
```
Comment (LiteSun) [inline web/src/pages/Route/components/Step1/LabelsDrawer.tsx:27]: ?

File: web/src/pages/Route/components/Step1/MetaView.tsx [hunk 2]
```diff
@@ -14,21 +14,88 @@
...
+          </Form.Item>
+        )}
+      </React.Fragment>
+    );
+  };
+
+  const VersionLabelComponent = () => {
+    return (
+      <React.Fragment>
+        <Form.Item label="版本" name="custom_version_selected_label">
```
Comment (LiteSun) [inline web/src/pages/Route/components/Step1/MetaView.tsx:90]: i18n

File: web/src/pages/Route/components/Step1/LabelsDrawer.tsx [hunk 3]
```diff
@@ -108,12 +112,14 @@ const LabelList = (disabled: boolean, labelList: RouteModule.LabelList) => {
 };
 
 const LabelsDrawer: React.FC<Props> = ({
-  disabled,
-  labelsDataSource,
+  title = 'Label Manager',
```
Comment (liuxiran) [inline web/src/pages/Route/components/Step1/LabelsDrawer.tsx:115]: i18n

Comment (juzhiyuan) [reply web/src/pages/Route/components/Step1/LabelsDrawer.tsx:115]: Will update in another PR :)

Comment (juzhiyuan) [reply web/src/pages/Route/components/Step1/LabelsDrawer.tsx:115]: *i18n

**Review by liuxiran — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**